aws/dual-region/scripts/generate_zeebe_helm_values.sh

```diff
@@ -52,23 +77,38 @@ if [[ "$namespace_0" == "$namespace_1" ]]; then
     exit 1
 fi
 
+if [ "$mode" = "failover" ] && [ "$lost_region" -ne 0 ] && [ "$lost_region" -ne 1 ]; then
+    echo "Invalid region $lost_region provided for the lost region. Please provide either 0 or 1 as input value."
+    exit 1
+fi
+
 # Generating and printing the string
 initial_contact=$(generate_initial_contact "$namespace_0" "$namespace_1" "$helm_release_name" "$cluster_size")
```

Comment:
to double check: should this always be the same values (no brokers in the failover namespaces), even in the `failover` mode?

Reply:
yes, it's always supposed to be the same.
It's only about the initial Zeebe cluster creation / forming. After that if a broker says hallo, it can simply join it.
So in the case of failover, the brokers will know half the contact points and can ping them to join the Zeebe cluster.